### ISSUE TITLE
How legal is stealing Boxrec's data?

### ISSUE THREAD

**Issue by mjwhaley** [boxing/boxrec#78]
Assuming that the data that is being pulled using this code isn't being done with Boxrec's consent. How legal is this?

**Comment by mikedidomizio** [comment]
The term for what this falls under is [Web scraping](https://en.wikipedia.org/wiki/Web_scraping).  Scraping has always been grey area, and what you do with the information can change that.  This project fetches a web page, parses it looking for keywords and the data associated with it, returning it in a usable format, and that's where it stops.

The terms and conditions for BoxRec can be found [here](http://boxrec.com/en/terms_conditions).  The general section outlines what you may and may not do in regards to BoxRec data.

tl;dr this is legal.  There are scenarios where you could use this illegally though.

**Comment by mikedidomizio** [comment]
The BoxRec terms and conditions have changed.  They state:

> The use of automated systems or software to extract data from boxrec.com (‘screen scraping’) is prohibited unless the third party has directly concluded a written licence agreement with BoxRec in which permits it access to BoxRec’s information.

So my revised answer is: this package is not stealing the data but it is against their ToS to use this package against their website.  Although the package already states it is not responsible for the use of it, the README will be getting a disclaimer noting where BoxRec stands on the use of scrapers.

I'm going to close this issue.  If something changes or you have more to add, feel free to re-open this or create a new issue.
